packages/next/src/server/app-render/action-handler.ts

```diff
@@ -798,7 +798,7 @@ export async function handleAction({
             decodeFormState,
           } = require(
             `./react-server.node`
-          ) as typeof import('./react-server.node')
+          ) as typeof import('react-server-dom-webpack/server.node')
```

Comment:
This should've issue a lint error since `require(source)` and `import(source)` mismatch

Reply:
fixed in the lint rule

but that leaves this problem
<img width="1680" height="141" alt="Bildschirmfoto 2025-08-04 um 14 15 20" src="https://github.com/user-attachments/assets/aa406539-af67-4ab1-a8da-acd7cb90b7f0" />